app/furniture/marketplace/carts/_footer.html.erb

```diff
@@ -8,7 +8,7 @@
   <tr>
     <td class="text-left px-3 py-3.5 text-right" colspan="8">
       <%= render "marketplace/carts/total", cart: cart %>
-      <%- if cart.delivery_address.present? %>
+      <%- if cart.delivery_address.present? && cart.contact_phone_number.present?%>
```

Comment:
It may be nice to have a `cart.ready_for_delivery?` or something that contains these checks